build_release.py

```
@@ -109,7 +109,7 @@ def _abi_encode(types, values):
 
 deploymentInfo = {"dataContracts": [], "logicContracts": []}
 
-if metadata is not None:
+if metadata is not None and config.get("dataContracts"):
     for deployment, code in zip(config["dataContracts"], metadata["bytecodes"]):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)